course/frontend/prototypes-2/scripts/main.js

```diff
@@ -1,94 +1,64 @@
-/* Person
-a) Write a constructor function called Person and that has the following arguments: */
+var fxx = new Ferrari('fxx', 5000);
```

Comment:
PROBLEM here! why is Car / ferrari here? they belong to a demo code brought on screen at "prototypes" folder, but we are in prototypes-2 here, the Person vs Student vs Teacher challenge. JUST push the code corresponding to that, but BEWARE not replacing already existing code. MAY a develop reset be required to pull your develop branch back to original state in upstream first, before adding your changes (never deleting solutions provided before by other people).